packages/ui-components/src/components/Message/Message.component.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Juno contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useEffect, useState, useRef } from "react"
+
+import { Icon } from "../Icon"
+
+import { MessageProps } from "./Message.types"
+import { getMuiIcon } from "./utils/iconUtils"
```

Comment:
I don't see a big advantage of moving the icon logic into utils if only this one component makes use of it. However with small adjustments it might be reusable for all components that use icons. So instead of checking `MessageType` make a more generic type like `SemanticVariant` or something, then move the iconUtils up a few levels to `src/utils`. 

Though my suggestion would be to not do the generalization now but rather do this in a new PR and then use this new shared util in all components with icons. Though it might be more useful to do this after everything (or at least all components that use icons) have moved to Typescript :)

Reply:
I didn't want to touch the other components and just try it out here. 🙈 But I'll revert all externalisation and we can discuss it in the future. Thank You.